harpy/common/misc.py

```
@@ -0,0 +1,113 @@
+"""Module with helper function to set up Harpy workflows"""
+
+import os
+import sys
+import glob
+import gzip
+import shutil
+import yaml
```

Ruff (0.12.2)

`yaml` imported but unused

Remove unused import: `yaml`

(F401)